### PR TITLE
feat(cli): add `app delete` command

### PR DESCRIPTION
**Pull request by jfeodor**
Adds the command `numerous app delete` which deletes an app from Numerous, all associated information, and stops all related workloads.

### CODE REVIEW

File: cli/internal/app/app_delete_test.go [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+		input := DeleteAppInput{
+			OrganizationSlug: "organization-slug",
+			Name:             "app-name",
+		}
+		err := s.Delete(context.TODO(), input)
+
+		assert.ErrorIs(t, err, gql.ErrAccesDenied)
+	})
+
+	t.Run("given graphql error then it returns access denied error", func(t *testing.T) {
```
Comment (moroderNumerous) [inline cli/internal/app/app_delete_test.go:41]:
```suggestion
	t.Run("given graphql error then it returns given graphql error", func(t *testing.T) {
```

Comment (moroderNumerous) [reply cli/internal/app/app_delete_test.go:41]: I don't care what you change it to, It's just clear that's it's a leftover from a copy paste 😄

**Review by moroderNumerous — CHANGES_REQUESTED** [review]

File: cli/cmd/app/deletecmd/cmd.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
+package deletecmd
```
[the remark this answers is not in the document]
Comment (jfeodor) [reply cli/cmd/app/deletecmd/cmd.go:1]: `delete` is a reserved word so I had to find an alternative :smile:

File: cli/cmd/app/deletecmd/cmd.go [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+var (
+	slug    string
+	appName string
+	appDir  string = "."
+)
+
+func run(cmd *cobra.Command, args []string) {
+	service := app.New(gql.NewClient(), nil, http.DefaultClient)
+
+	if err := Delete(cmd.Context(), service, appDir, slug, appName); err != nil {
```
[the remark this answers is not in the document]
Comment (jfeodor) [reply cli/cmd/app/deletecmd/cmd.go]: nice idea :smile: 489ee60

File: cli/cmd/app/appident/get_appidentifier.go [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+import (
+	"path/filepath"
+
+	"numerous/cli/cmd/output"
+	"numerous/cli/cmd/validate"
+	"numerous/cli/manifest"
+)
+
+// Uses the given slug and appName, or loads from manifest, and validates.
+func GetAppIdentifier(appDir string, slug string, appName string) (string, string, error) {
```
[the remark this answers is not in the document]
Comment (jfeodor) [reply cli/cmd/app/appident/get_appidentifier.go]: Done in 412a84e

**Review by moroderNumerous — APPROVED** [review]

**Pull request merged**